### ISSUE TITLE
New website - what still lacks

### ISSUE THREAD

**Issue by Ja-Tar** [Ja-Tar/AktualneInstrukcjePKP#25]
Needs to have the same functionality as original, plus:

- correct WCAG functionality
- instruction history
- backups of old versions (most important instructions)
